error_post_go120.go

```diff
@@ -27,3 +27,37 @@ package multierr
 func (merr *multiError) Unwrap() []error {
 	return merr.Errors()
 }
+
+type multipleErrors interface {
+	Unwrap() []error
+}
+
+// Errors returns a slice containing zero or more errors that the supplied
+// error is composed of. If the error is nil, a nil slice is returned.
+//
+//	err := multierr.Append(r.Close(), w.Close())
+//	errors := multierr.Errors(err)
+//
+// If the error is not composed of other errors, the returned slice contains
+// just the error that was passed in.
+//
+// Callers of this function are free to modify the returned slice.
+func Errors(err error) []error {
```

Comment:
Instead of duplicating the doc string and risk having it change between versions, consider keeping this in error.go, and adding an unexported helper in the version-specific files.

```
// error.go

// Errors returns a slice [..]
func Errors(err error) []error {
  return extractErrors(err)
}

// error_post_go120.go
func extractErrors(err error) []error { ... }
```